oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -111,3 +111,13 @@ message InvokeResponse {
   // TODO(#3843): Use explicit `oak_crypto` messages.
   bytes encrypted_body = 1;
 }
+
+message GetEncryptionKeyRequest {
+  // Encrypted attestation evidence of the dependent enclave.
+  bytes encrypted_attestation_bundle = 1;
+}
+
+message GetEncryptionKeyResponse {
+  // Encrypted attestation evidence, public key, and private key of the leader enclave.
```

Comment:
The evidence and the public key should not be encrypted, since they will be processed by the launcher.

So there could just be 2 fields:
- `endorsed_evidence` (bundle for now)
- and the `encrypted_encryption_key`